### ISSUE TITLE
Remove "teckentolkat" from -A

### ISSUE THREAD

**Issue by yabbah** [spaam/svtplay-dl#1025]
Is it possible to be able to download everything (woth -A) without the "teckentolkat"-episodes mostly from svtplay.se?
/yabbah

**Comment by mbjurstrom** [comment]
Try with --exclude teckentolkat

**Comment by yabbah** [comment]
oh, thanks. Totally missed that lovely feature.

**Comment by Sopor-** [comment]
@yabbah add `--exclude=syntolkat,teckentolkat` to be sure you skip both...

**Comment by granbom** [comment]
Also `--exclude=tolkat` will work.
And at least I think, the best solution (then you don't have to write it in the command line at all, is to add
```
service:
  svtplay:
    exclude: tolkat
```
in the [.svtplay-dl.yaml](https://svtplay-dl.se/config/) config file

**Comment by Sopor-** [comment]
@granbom If you add `tolkat` you will not be able to download `https://www.svtplay.se/video/19852521/babel-poppoesi/babel-poppoesi-sasong-2-jenny-wilson-tolkar-karin-boye`

**Comment by Sopor-** [comment]
@granbom I prefer to use the whole name incase some of the file names use `tolkat` :)

**Comment by granbom** [comment]
@Sopor- true, but I found it highly unlikely that a program will be named `tolkat`. 
Tolkar, tolkade, tolka would be possible. On the other hand, then there might be a program called: `SVT-dokumentär om personerna bakom syntolkat` too.
The only way I would se tolkat might be used, would be something like `Linda Olofsson har tolkat med hundspann till Kebnekajse`. But I think they would use present tense and `Mitt i naturen tolkar med hundspann till Kebnekase` in that case.
Regardless I agree that one should have in mind that you might exclude something you wanted in any case. If so, temporarily just add a `#` on the exclude line :)

**Comment by psonnosp** [comment]
This doesn't work any longer, atleast not for me on Win10 and 3.3.
I've tried both --exclude=syntolkat in CMD and as a yaml as above. The other stuff in the yaml work but also the syntolkat stuff that I try to exclude. Maybe SVT has changed something?

**Comment by Sopor** [comment]
Any example URL @psonnosp ...

**Comment by psonnosp** [comment]
Sure, when I try --exclude=syntolkat -A https://www.svtplay.se/festen I get 60 episodes instead of the expected 30.
But I just now tried it on https://www.svtplay.se/snoanglar and there it worked.

**Comment by Sopor** [comment]
It will show 60 but it will skip those that match `--exclude=syntolkat`.
